xchange-latoken/src/main/java/org/knowm/xchange/latoken/LatokenAdapters.java

```diff
@@ -0,0 +1,266 @@
+package org.knowm.xchange.latoken;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.Order.OrderStatus;
+import org.knowm.xchange.dto.Order.OrderType;
+import org.knowm.xchange.dto.account.Balance;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+import org.knowm.xchange.dto.marketdata.Ticker;
+import org.knowm.xchange.dto.marketdata.Trade;
+import org.knowm.xchange.dto.marketdata.Trades;
+import org.knowm.xchange.dto.marketdata.Trades.TradeSortType;
+import org.knowm.xchange.dto.meta.CurrencyPairMetaData;
+import org.knowm.xchange.dto.trade.LimitOrder;
+import org.knowm.xchange.dto.trade.OpenOrders;
+import org.knowm.xchange.dto.trade.UserTrade;
+import org.knowm.xchange.dto.trade.UserTrades;
+import org.knowm.xchange.exceptions.ExchangeException;
+import org.knowm.xchange.latoken.dto.account.LatokenBalance;
+import org.knowm.xchange.latoken.dto.exchangeinfo.LatokenCurrency;
+import org.knowm.xchange.latoken.dto.exchangeinfo.LatokenPair;
+import org.knowm.xchange.latoken.dto.marketdata.LatokenOrderbook;
+import org.knowm.xchange.latoken.dto.marketdata.LatokenTicker;
+import org.knowm.xchange.latoken.dto.marketdata.LatokenTrade;
+import org.knowm.xchange.latoken.dto.marketdata.LatokenTrades;
+import org.knowm.xchange.latoken.dto.trade.LatokenOrder;
+import org.knowm.xchange.latoken.dto.trade.LatokenOrderSide;
+import org.knowm.xchange.latoken.dto.trade.LatokenOrderStatus;
+import org.knowm.xchange.latoken.dto.trade.LatokenUserTrade;
+import org.knowm.xchange.latoken.dto.trade.LatokenUserTrades;
+
+public class LatokenAdapters {
+
+  protected static List<LatokenPair> allPairs;
```

Comment:
this type of static attributes will fail in multi threaded applications and should not be used
if you need to store some results from remoteInit then store it in an Exchange instance ad just pass it here every time you need it